### PR TITLE
Ui widgets

### PR DESCRIPTION
**Pull request by DamnBack**

### CODE REVIEW

File: threedi_api_qgis_client/widgets/wizard.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+uicls_p4, basecls_p4 = uic.loadUiType(os.path.join(base_dir, 'ui', 'page4.ui'))
+
+
+def set_widget_background_color(widget, hex_color='#F0F0F0'):
+    palette = widget.palette()
+    palette.setColor(widget.backgroundRole(), QColor(hex_color))
+    widget.setPalette(palette)
+
+
+class Page1Widget(uicls_p1, basecls_p1):
```
Comment (PeterPetrik) [inline threedi_api_qgis_client/widgets/wizard.py]: Can you maybe rename Page[X] to something more explanatory like PageFinalSummaryWidget, PagePrecipitationWidget...

File: threedi_api_qgis_client/widgets/wizard.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+from qgis.PyQt.QtGui import QColor
+
+from qgis.PyQt.QtWidgets import QWizardPage, QWizard, QGridLayout, QGraphicsScene, QSizePolicy, QInputDialog
+try:
+    import pyqtgraph as pg
+except ImportError:
+    main_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    deps_path = os.path.join(main_dir, "deps", "pyqtgraph-0.10.0-py3-none-any.whl")
+    sys.path.append(deps_path)
+    import pyqtgraph as pg
```
Comment (PeterPetrik) [inline threedi_api_qgis_client/widgets/wizard.py]: I would put this wrapper to the separate file like ./custom_imports.py, so we can use it from multiple files

File: threedi_api_qgis_client/widgets/wizard.py (hunk 1 above)
Comment (DamnBack) [reply threedi_api_qgis_client/widgets/wizard.py]: Done.

File: threedi_api_qgis_client/widgets/wizard.py (hunk 2 above)
Comment (DamnBack) [reply threedi_api_qgis_client/widgets/wizard.py]: Done.

**Pull request merged**